app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
Re: the `seeds.rb` question, we definitely want to keep the file. If it were me, the only things I would put in there are the root division, and *maybe* the countries, languages, and currencies (but without hardcoding their `id:` attributes, if at all possible.)

I think everything else is/should be more or less optional, if the system won't crash because they're gone, it probably doesn't need to be there, but there are occasional exceptions and reasons to compromise on that point.